internal/common/foraging/deerhunt.go

```diff
@@ -0,0 +1,67 @@
+package foraging
+
+// see https://colab.research.google.com/drive/1g1tiX27Ds7FGjj4_WjFB3OLj8Fat_Ur5?usp=sharing for experiments + simulations
+
+import (
+	"math"
+
+	"github.com/SOMAS2020/SOMAS2020/internal/common/shared"
+	"gonum.org/v1/gonum/stat/distuv"
+)
+
+type DeerHuntParams struct {
+	P   float64 // Bernoulli p variable (whether or not a deer is caught)
```

Comment:
if you leave your comment above the lines instead, they will appear in intellisense when you hover :D